<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuLugares</class>
 <widget class="QDialog" name="MenuLugares">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 195, 221);</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>741</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_lugares">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>739</width>
      <height>399</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_null">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;DejaVu Sans Mono&quot;;
color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string>No se encontraron lugares</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_atras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Atr√°s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tema">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>109</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Pictures/bigfood.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_tematexto">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>401</width>
     <height>109</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jamrul</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;Jamrul&quot;;
color: rgb(73, 60, 107);</string>
   </property>
   <property name="text">
    <string>COMIDA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Pictures/turr.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_costo1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconoprecio.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_costo2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconoprecio.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_costo3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconoprecio.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_costo3</zorder>
   <zorder>label_costo2</zorder>
   <zorder>label_costo1</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>147</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_estrella1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconocalif.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_estrella2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconocalif.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_estrella3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconocalif.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_estrella4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconocalif.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_estrella2</zorder>
   <zorder>label_estrella1</zorder>
   <zorder>label_estrella3</zorder>
   <zorder>label_estrella4</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>218</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconoubicacion.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_lugar">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tlaquepaque</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
